arda/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "main": "js/app.js",
+  "scripts": {
+    "start": "../node_modules/watchify/bin/cmd.js . -o build/bundle.js -v -d",
+    "build": "browserify js/app.js > build/bundle.js"
+  },
+  "browserify": {
+    "transform": [
+      [
+        "babelify",
+        {
+          "experimental": true,
+          "optional": [
+            "runtime"
+          ]
+        }
+      ],
+      "envify"
+    ]
+  },
+  "dependencies": {
+    "arda": "^0.13.1"
```

Comment:
@mizchi please add `arda` to global dependencies instead - https://github.com/voronianski/flux-comparison/blob/master/package.json#L14